src/client.coffee

```diff
@@ -20,17 +20,22 @@ parser = new xml2js.Parser
     default_host {String} Optional
   Example:
     Client = require('3scale').Client
-    client = new Client(provider_key, [default_host])
+    client = new Client(provider_key, [default_host], [default_port])
 ###
 
+host = process.env.host || "su1.3scale.net";
+port = process.env.port || 443;
```

Comment:
I think it's a bit dangerous to use these env vars, as they may be used for other purposes (for the PORT and HOST of the API app itself, for example). Maybe we need something more specific? (`threescale_host`, `threescale_port` or similar)